backend/LexBoxApi/Services/MySqlMigrationService.cs

```diff
@@ -93,6 +93,26 @@ private async Task Migrate(RedmineDbContext dbContext, DateTimeOffset now)
             {
                 user.LastActive = rmUser.LastLoginOn?.ToUniversalTime() ?? default(DateTimeOffset);
             }
+            if (user.CreatedDate > rmUser.CreatedOn)
```

Comment:
Is there any value in filtering a bit more. E.g.
```suggestion
            if (rmUser.CreatedOn is not null && rumUser.CreatedOn != default(DateTimeOffset) && user.CreatedDate > rmUser.CreatedOn)
```

Reply:
no because `date > null` is false and CreatedOn will never be default because it's nullable.